MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+# Maintainers of Limitador
+
+| Name                | Company  | GitHub          |
+|---------------------|----------|-----------------|
+| Eguzki Astiz Lezaun | Red Hat  | eguzki          |
+| Alex Snaps          | Red Hat  | alexsnaps       |
+| Didier Di Cesare    | Red Hat  | didierofrivia   |
+| Kevin Fan           | Red Hat  | KevFan          |
```

Comment:
```suggestion
| Alex Snaps          | Red Hat  | alexsnaps       |
| Didier Di Cesare    | Red Hat  | didierofrivia   |
| Eguzki Astiz Lezaun | Red Hat  | eguzki          |
| Kevin Fan           | Red Hat  | KevFan          |
```

Reply:
Thanks @guicassolato - I've addressed this now